app/services/store.js

```diff
@@ -68,6 +69,20 @@ export default DS.Store.extend({
       this.push(this.normalize('commit', commit));
     }
 
+    if (name === 'branch') {
+      // force reload of repo branches
+      // delay to resolve race between github-sync and live
+      Ember.run.later(() => {
+        const repo = this.recordForId('repo', data.repository_id);
+        repo.notifyPropertyChange('branches');
+        repo.get('branches').then(branches => {
+          this.set('branches.amount', branches.content.length);
```

Comment:
Services are singletons, so this would set amounts of branches regardless of a repository that the branch was created for, I'm not sure if that's what you intended to do here.